Example/Example/EditorDemoController.swift

```diff
@@ -993,6 +993,8 @@ extension EditorDemoController {
         }
 
         toolbar.overflowToggleIcon = Gridicon.iconOfType(.ellipsis)
+        toolbar.overflowToggleAccessibilityLabel = NSLocalizedString("More", comment: "Accessibility label for the More button on formatting toolbar.")
```

Comment:
I was thinking, while it's good to allow this to be configurable from the outside shouldn't we have a default value for the label inside the library code?

Reply:
That's a good idea. I'm not familiar with how localizations work for libraries. Is using `NSLocalizedString` like here enough?

Reply:
Based on [WPiOS's `Localizable.strings`](https://github.com/wordpress-mobile/WordPress-iOS/blob/44a4dc6bf1bb44b677293faacf86e761cd7255e8/WordPress/Resources/ja.lproj/Localizable.strings#L5706-L5707), it looks like the `NSLocalizedString()` calls in Aztec are copied over to it. I initially thought they weren't. It looks like we don't need to have the customizability after all. 

- Moved the `NSLocalizedString()` to the `overflowToggleItem` 82208f1d6ff6ce99dcef738a58722cc3712d9fa6
- Removed unnecessary property f2d46f5960886eaf839d5b5c7cd7036d76b66091

Thank you for pointing this out, @SergioEstevao. 🙂